assets/queries/ansible/aws/cloudformation_in_use/test/positive_expected_result.json

```diff
@@ -0,0 +1,25 @@
+[
+	{
+		"queryName": "AWS CloudFormation In Use",
+		"severity": "MEDIUM",
+		"line": 2
+	},
+
+	{
+		"queryName": "AWS CloudFormation In Use",
+		"severity": "MEDIUM",
+		"line": 15
+	},
+
+	{
+		"queryName": "AWS CloudFormation In Use",
+		"severity": "MEDIUM",
+		"line":  30
+	},
+
+	{
+		"queryName": "AWS CloudFormation In Use",
+		"severity": "MEDIUM",
+		"line":  40
+	}
+]
```

Comment:
Change query name to "AWS CloudFormation Without Template", to be more specific and better represent the vulnerability. Do the same in the corresponding Terraform query, please.